uws/UWS/base.py

```diff
@@ -38,6 +38,7 @@ def get_job(self, id):
             raise UWSError("Malformatted response. Are you sure the host you specified is a IVOA UWS service?", raw)
         except Exception as e:
             raise e
+            # TODO: had problems with "raise e" here: AttributeError: 'NoneType' object has no attribute 'text'
```

Comment:
can you paste the complete stack trace here? This is not an issue with the raise, but a bug further up the stack